src/client/AppState.ts

```diff
@@ -103,6 +103,8 @@ export const PRISTINE_APPSTATE: AppState = {
   ),
 
   HORECA: apiPristineResult([]),
+
+  AVG: apiPristineResult(null),
```

Comment:
Hier nog een Feature Toggle bij?

```suggestion
  AVG: apiPristineResult(null,{ isActive: FeatureToggle.avgActive }),
```